packages/modular-scripts/src/build.ts

```diff
@@ -148,6 +148,8 @@ const typescriptConfig: TSConfig = {};
     throw new Error('Failed to load Typescript configuration');
   }
 
+  const exclude: string[] = (configObject.exclude as string[]) || [];
```

Comment:
Could you add a comment here why this is necessary?

Reply:
Done